examples/source_examples/aws_athena/using_boto3/connector.py

```diff
@@ -0,0 +1,128 @@
+                row_data.append(int(cell['BigIntValue']))
+            elif 'DoubleValue' in cell:
+                row_data.append(float(cell['DoubleValue']))
+            elif 'BooleanValue' in cell:
+                row_data.append(bool(cell['BooleanValue']))
+            else:
+                row_data.append(None)
+        yield op.upsert(table="customers",
+                        data={
+                            "customer_id": row_data[0],  # Customer id.
+                            "first_name": row_data[1],  # First Name.
+                            "last_name": row_data[2],  # Last name.
+                            "email": row_data[3],  # Email id.
+                        })
+
+
+def get_query_results(athena_client, query_execution_id):
+    response = athena_client.get_query_results(QueryExecutionId=query_execution_id)
+
+    while True:
+        response = athena_client.get_query_results(QueryExecutionId=query_execution_id,
+                                                   NextToken=response[NEXT_TOKEN]) \
+            if NEXT_TOKEN in response else athena_client.get_query_results(QueryExecutionId=query_execution_id)
+
+        yield from process_and_upsert_response(response)
+        if NEXT_TOKEN not in response:
+            break
+
+# Define the update function, which is a required function, and is called by Fivetran during each sync.
+# See the technical reference documentation for more details on the update function
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update
+# The function takes two parameters:
+# - configuration: dictionary contains any secrets or payloads you configure when deploying the connector
+# - state: a dictionary contains whatever state you have chosen to checkpoint during the prior sync
+# The state dictionary is empty for the first sync or for any full re-sync
+def update(configuration: dict, state: dict):
+    athena_client = create_athena_client(configuration)
+
+    query = f"SELECT * FROM {TABLE_NAME}"
+    response = athena_client.start_query_execution(
+        QueryString=query,
+        QueryExecutionContext={'Database': configuration['database_name']},
+        ResultConfiguration={
+            'OutputLocation': configuration['s3_staging_dir']
+        }
+    )
+
+    query_execution_id = response['QueryExecutionId']
+
+    # Wait for query completion (adjust polling interval as needed)
+    while True:
+        response = athena_client.get_query_execution(QueryExecutionId=query_execution_id)
+        status = response['QueryExecution']['Status']['State']
+        if status in ['SUCCEEDED', 'FAILED', 'CANCELLED']:
+            break
+
+        time.sleep(5)
+
+    if status == 'SUCCEEDED':
+        yield from get_query_results(athena_client, query_execution_id)
+    else:
+        print(f"Query failed with status: {status}")
+
+
+# This creates the connector object that will use the update function defined in this connector.py file.
```

Comment:
```suggestion
# This creates the connector object - connection - that will use the update function defined in this connector.py file.
```

Reply:
This is incorrect. When we say a connector object we mean the Connector Object in python code referred in the sdk package